src/murmur/Messages.cpp

```diff
@@ -863,12 +863,6 @@ void Server::msgChannelState(ServerUser *uSource, MumbleProto::ChannelState &msg
 			return;
 		}
 
-		QRegExp re2("\\w");
-		if (re2.indexIn(qsName) == -1) {
```

Comment:
Nothing *should* interpret an empty channel string as something though, right?
Fair enough. I’ll add a 1 char requirement/check.